packages/db/src/translate/granules.ts

```diff
@@ -87,7 +88,7 @@ export const translatePostgresGranuleToApiGranule = async ({
     duration: granulePgRecord.duration,
     endingDateTime: granulePgRecord.ending_date_time?.getTime().toString(),
     error: granulePgRecord.error,
-    execution: executionArns[0] ? executionArns[0].arn : undefined,
+    execution: executionArns[0] ? getExecutionUrlFromArn(executionArns[0].arn) : undefined,
```

Comment:
This was a problem for a few integration tests. It looks like the `execution` property on granules was the full URL, not just the ARN. From the schema:

```
execution: {
      title: 'Step Function Execution link',
      type: 'string',
      readonly: true,
    },
```